src/chainlit/langchain/callbacks.py

```diff
@@ -375,6 +385,24 @@ def on_llm_end(self, response: LLMResult, **kwargs: Any) -> None:
     def on_chain_start(
         self, serialized: Dict[str, Any], inputs: Dict[str, Any], **kwargs: Any
     ) -> None:
+        path = serialized.get("kwargs", {}).get("llm", {}).get("id", [])
```

Comment:
what is path exactly?

Reply:
it is the full path of the class in the langchain module, it is used as an ID now (more on that in https://github.com/Chainlit/chainlit/pull/312/files#r1304264859)